.github/workflows/deploy_with_cdk.yml

```diff
@@ -9,14 +12,17 @@ on:
         required: true
       private_gh_packages_token:
         required: true
-      aws_access_key_id:
-        required: true
-      aws_secret_access_key:
-        required: true
     inputs:
       region:
```

Comment:
I think it makes sense to remove the inconsistencies in the use of `region` vs. `aws-region` at this time, since we're going to be updating all deployments in other projects.

Reply:
So would you rather use `region` or `aws-region`? Do I need to change something here?